onnxconverter_common/optimizer.py

```diff
@@ -723,6 +724,73 @@ def _build_onnx_model(node_list):
     return regenerated
 
 
+def _visit(name_to_node_map, n_name, result):
+    node = name_to_node_map[n_name]
+    if node.status == 'perm':
```

Comment:
a boolean variable seems enough for this case, None, False, True.

Reply:
Using string can be more explicitly read.